apis/Google.Cloud.Firestore/Google.Cloud.Firestore/WriteBatch.cs

```diff
@@ -264,7 +244,9 @@ internal async Task<IList<WriteResult>> CommitAsync(ByteString transactionId, Ca
                     Fields = { fields },
                     Name = documentReference.Path,
                 },
-                UpdateMask = includeEmptyUpdateMask || updatePaths.Count > 0 ? new DocumentMask { FieldPaths = { updatePaths.Select(fp => fp.EncodedPath) } } : null,
+                UpdateMask = updatePaths is null
```

Comment:
I could compute the DocumentMask (or null) outside the object initializer if you'd prefer.

Reply:
No, this is fine.